README.rst

```diff
@@ -0,0 +1,171 @@
+that is required to bring the KNoT platform on RaspberryPi.
+
+**Warning**: You will need to have access to the network, since Buildroot will
+download the packages' sources.
+
+You should base your work on this defconfig:
+
+#. Run KNoT defconfig
+
+   .. code-block:: bash
+
+      make knot_<board>_defconfig
+
+   .. note::
+      Currently supported KNoT defconfigs:
+
+         - ``knot_raspberrypi3_defconfig``
+         - ``knot_raspberrypi0w_defconfig``
+
+#. Menu Configuration
+
+   You can access and select extra packages you wish to compile with KNoT Linux
+   OS on Buildroot menuconfig.
+
+   .. code-block:: bash
+
+      make menuconfig
+
+#. Build
+
+   .. code-block:: bash
+
+      make
+
+   **Tip**: This may take a while. Consider getting yourself a coffee ;-)
+
+
+Result of the build
+-------------------
+
+After building, you should obtain this tree:
+
+.. code-block:: text
+
+   output/images/
+   ├── bcm2708-rpi-b.dtb           [1]
+   ├── bcm2708-rpi-b-plus.dtb      [1]
+   ├── bcm2708-rpi-0-w             [1]
+   ├── bcm2709-rpi-2-b.dtb         [1]
+   ├── bcm2710-rpi-3-b.dtb         [1]
+   ├── bcm2710-rpi-3-b-plus.dtb    [1]
+   ├── boot.vfat
+   ├── rootfs.ext4
+   ├── rpi-firmware/
+   |   ├── bootcode.bin
+   |   ├── cmdline.txt
+   |   ├── config.txt
+   |   ├── fixup.dat
+   |   ├── start.elf
+   |   └── overlays/               [2]
+   ├── sdcard.img
+   └── zImage
+
+[1] Not all of them will be present, depending on the RaspberryPi model
+you are using.
+
+[2] Only for the Raspberry Pi 3 Model (overlay pi3-miniuart-bt is needed
+to enable the RPi3 serial console otherwise occupied by the bluetooth
+chip). Alternative would be to disable the serial console in cmdline.txt
+and /etc/inittab.
+
+----------------------------------------------------------------
+
+How to write the SD card
+------------------------
+
+Once the build process is finished you will have an image called "sdcard.img"
+in the output/images/ directory.
+
+It is possible to erase and write the SD Card using native OS application to
+manage and configure disk drives. Similarly to **Disks** on Ubuntu.
+Another way is using ``dd`` command on a linux terminal.
+
+Copy the bootable "sdcard.img" onto an SD card with "dd":
+
+   .. code-block:: bash
+
+      sudo dd if=output/images/sdcard.img of=/dev/sdX
+
+   **Tip**: See ``dd`` manual for more options. Like to see the progress you can add ``status=progress`` in the end of above ``dd`` command.
+
+After a successful flash, insert the SD Card into your Raspberry Pi, and power
+it up.
+
+----------------------------------------------------------------
+
+Monitoring the KNoT Gateway
+---------------------------
+
+You can access you gateway using ``ssh`` command.
+
+In a linux machine on same local network and execute the followed command
+on terminal:
+
+.. code-block:: bash
+
+   arp-scan -l | grep b8:27:eb
+
+**Note**: It is possible needed to install ``arp-scan``. To do this running
```

Comment:
There are some concordance issues here.  Also, `apt-get` is for Ubuntu/Debian.

suggestion:
**Note**: It is necessary to have ``arp-scan`` package installed. If you're running a Ubuntu/Debian based distro, run ``sudo apt-get install arp-scan`` on terminal.